pkg/collector/corechecks/network-devices/cisco-sdwan/client/request.go

```diff
@@ -82,7 +82,7 @@ func (client *Client) get(endpoint string, params map[string]string) ([]byte, er
 		}
 	}
 
-	return nil, fmt.Errorf("http responsed with %d code", statusCode)
+	return nil, fmt.Errorf("%s http responsed with %d code", endpoint, statusCode)
```

Comment:
Unrelated language nit 😛 
```suggestion
	return nil, fmt.Errorf("%s http responded with %d code", endpoint, statusCode)
```

Reply:
thanks!